compiler/src/dotty/tools/dotc/core/Mode.scala

```diff
@@ -87,7 +90,7 @@ object Mode {
   /** Don't suppress exceptions thrown during show */
   val PrintShowExceptions = newMode(18, "PrintShowExceptions")
 
-  val PatternOrType = Pattern | Type
+  val PatternOrTypeBits = Pattern | Type | InPatternAlternative
```

Comment:
Why did you rename this value? I think the old name is more consistent with the other mode's name

Reply:
It's because otherwise we'd risk that someone thinks it's `Pattern | Type` only. And `PatternOrTypeorInPatternAlternative` was too long.